Cheza-Dzabala/idove_mobile
# Implement Navigation UI

### Description
This PR implements the user interface for navigating through the application. 

It adds the following:
- Home page
- Backdrop for main navigation
- Bottom Navigation Bar

### Screenshots (if applicable)
***insert screenshots of the work done***

### Type of Change



### Checklist
- [ ] This PR has been tested


- [x] This PR meets all the client standards & conventions

### How To Manually Test

> Clone this repository
> Run pub get
> Deploy to emulator or device
> Ensure that description and screenshots match up to required standards
